src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.db.InputFileHandlerImpl;
+import core.basesyntax.db.InputFileReaderImpl;
+import core.basesyntax.db.ReportFileWriter;
+import core.basesyntax.db.ReportFileWriterImpl;
+import core.basesyntax.handlers.BalanceHandler;
+import core.basesyntax.handlers.OperationHandler;
+import core.basesyntax.handlers.PurchaseHandler;
+import core.basesyntax.handlers.QuantityAdditionHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Main {
+    private static final String INPUT_FILE_PATH = "jv-fruit-shop/src/main/resources/input.csv";
+    private static final String REPORT_FILE_PATH = "jv-fruit-shop/src/main/resources/report.csv";

```

Comment:
```suggestion
    private static final String INPUT_FILE_PATH = "src/main/resources/input.csv";
    private static final String REPORT_FILE_PATH = "src/main/resources/report.csv";
```
will this work?

Reply:
I have no idea, why it doesn't work in my IDE...